README.md

```diff
@@ -2,10 +2,12 @@
 
 This repository contains the code for NHS.UK Chart components
 
-These chart components are designed for visualizing Body Mass Index (BMI), Child Body Mass Index (CBMI), Blood Pressure (BP) and Waist to Height (WHTR) data.
+These chart components are designed for visualizing Body Mass Index (BMI), Child Body Mass Index (CBMI) and Blood Pressure (BP) data.
```

Comment:
It's worth rewriting this section and call out the components which this refers to

Reply:
resolved